doc/manuals/user/initial_notification.md

```diff
@@ -1,152 +1,109 @@
-# Initial notification
```

Comment:
Please, don't re-edit the whole file... It makes very hard to see the particular diference introduce in this PR. Please take back the original file and edit only the lines that need changes.

Reply:
fixed in 6769ef3